libc/test.yaml

```diff
@@ -85,7 +85,7 @@ test:
         value: [armv7a9-zynq7000-qemu]
     
     - name: unix-socket # Issue: 1342 | https://github.com/phoenix-rtos/phoenix-rtos-project/issues/1342
-      execute: test-libc-unix-socket --extra-poll-delay-ms 150 --transfer-loop-cnt 30
+      execute: test-libc-unix-socket --extra-poll-delay-ms 250 --transfer-loop-cnt 30
```

Comment:
About the commit message/description - please always look at the other commits in the repo, the format is as follows:

<img width="1385" height="388" alt="Screenshot from 2025-09-18 10-27-49" src="https://github.com/user-attachments/assets/b7cb40d0-d26d-4ef0-b420-f27fffc102f6" />

Reply:
So in your case add the newlines between the explanation. start from the uppercase and finish using dot. also I would extend this explanation a bit: `Increased due to bad qemu performance on this target.`